src/mongo/crypto/crypto_openssl.cpp

```diff
@@ -39,25 +39,45 @@
 #include <openssl/hmac.h>
 #include <openssl/sha.h>
 
+#include <cstring>
```

Comment:
I believe this is my final comment, and it is a minor one: cstring needs to be grouped with the openssl headers, without a line break.